client/views/main.js

```diff
@@ -16,3 +16,16 @@ Template.layout.rendered = function() {
 	$(window).resize(function(event){ getViewportWidth(); });
 	Session.set('isRetina', (window.devicePixelRatio == 2));
 };
+
+
+Template.layout.events({
+	'click .js-scroll': function(event, instance) {
+		var position = $(event.target.getAttribute('href')).offset();
+		if(typeof position != 'undefined') {
+			// subtract the amount of pixels of the height of the navbar
+			position = position.top - 50;
```

Comment:
Q: Can't we get the 50px as a variable out of a script? The top-navbar might not always be this size. Or define hight globally?